src/helper-modules/ordnance-survey/ordnance-survey.service.ts

```diff
@@ -0,0 +1,33 @@
+import { HttpService } from '@nestjs/axios';
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { KEY as ORDNANCE_SURVEY_CONFIG_KEY, OrdnanceSurveyConfig } from '@ukef/config/ordnance-survey.config';
+import { GEOSPATIAL } from '@ukef/constants';
+import { HttpClient } from '@ukef/modules/http/http.client';
+
+import { GetAddressesOrdnanceSurveyResponse } from './dto/get-addresses-ordnance-survey-response.dto';
+import { OrdnanceSurveyException } from './exception/ordnance-survey.exception';
+
+@Injectable()
+export class OrdnanceSurveyService {
+  private readonly httpClient: HttpClient;
+  private readonly key: string;
+
+  constructor(httpService: HttpService, configService: ConfigService) {
+    this.httpClient = new HttpClient(httpService);
+    const { key } = configService.get<OrdnanceSurveyConfig>(ORDNANCE_SURVEY_CONFIG_KEY);
+    this.key = key;
+  }
+
+  async getAddressesByPostcode(postcode: string): Promise<GetAddressesOrdnanceSurveyResponse> {
```

Comment:
Doc string?

Reply:
We don't use jsDocs in MDM. 
Also this function is quite short, purpose should be clear from function name and vs code already produces help for types using typescript type declarations.